<ui version="4.0" >
 <class>EncodeFile</class>
 <widget class="QWidget" name="EncodeFile" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>629</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <item>
    <widget class="QTreeWidget" name="fileList" >
     <property name="focusPolicy" >
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="itemsExpandable" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>File</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Title</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Artist</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Album</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Comment</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Genre</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Track</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Year</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Encoder</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QPushButton" name="addFilesButton" >
       <property name="focusPolicy" >
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text" >
        <string>Add Files</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton" >
       <property name="text" >
        <string>Clear List</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox1" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Selected Tracks</string>
       </property>
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="TextLabel5" >
          <property name="text" >
           <string>Artist:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="artist" />
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="TextLabel6" >
          <property name="text" >
           <string>Album:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="album" />
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="TextLabel7" >
          <property name="text" >
           <string>Year:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QSpinBox" name="year" >
          <property name="maximum" >
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="TextLabel1" >
          <property name="text" >
           <string>Genre:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QComboBox" name="genre" />
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="textLabel1" >
          <property name="text" >
           <string>Comment:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="1" >
         <widget class="QLineEdit" name="comment" />
        </item>
        <item row="5" column="1" >
         <widget class="QPushButton" name="assignButton" >
          <property name="text" >
           <string>Assign to selected tracks</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>468</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>artist</tabstop>
  <tabstop>album</tabstop>
  <tabstop>year</tabstop>
  <tabstop>genre</tabstop>
  <tabstop>comment</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
